quil-rs/src/instruction/calibration.rs

```diff
@@ -49,7 +51,7 @@ impl fmt::Display for Calibration {
             "DEFCAL {}{} {}:",
             self.name,
             parameter_str,
-            format_qubits(&self.qubits)
+            format_qubit_parameters(&self.qubits)
```

Comment:
This fixes a bug with how qubits being used in a `CalibrationDefinition` were printed. Per the Quil spec, variable qubits should be prefixed by a `%`.